packages/sdk/src/services/send.ts

```diff
@@ -0,0 +1,88 @@
+import { BigNumber, Signer } from 'ethers';
+import { PERIPHERY_ADDRESS } from '../utils/constants';
+import {
+  MakerTrade,
+  SettleTradeMaker,
+  SettleTradeTaker,
+  TakerTrade,
+} from '../utils/types';
+import { getGasBuffer } from '../utils/txHelpers';
+import {
+  encodeMakerOrder,
+  encodeSettlementMakers,
+  encodeSettlementTakers,
+  encodeTakerOrder,
+} from './encode';
+
+export type Transaction = {
+  from: string;
+  to: string;
+  data: string;
+  value?: string;
+};
+
+// todo: come up with a nice way of sending the owner address from the top level function
+
+export async function swap(trade: TakerTrade, chainId: number) {
+  const { calldata: data, value } = await encodeTakerOrder(trade);
+  await executeTransaction(trade.owner, data, value, chainId);
+}
+
+export async function makerOrder(trade: MakerTrade, chainId: number) {
+  const { calldata: data, value } = await encodeMakerOrder(trade);
+  await executeTransaction(trade.owner, data, value, chainId);
+}
+
+export async function settleMaker(trade: SettleTradeMaker, chainId: number) {
+  const { calldata: data, value } = encodeSettlementMakers(trade);
+  await executeTransaction(trade.owner, data, value, chainId);
+}
+
+export async function settleTaker(trade: SettleTradeTaker, chainId: number) {
+  const { calldata: data, value } = encodeSettlementTakers(trade);
+  await executeTransaction(trade.owner, data, value, chainId);
+}
+
+export async function executeTransaction(
+  signer: Signer,
+  data: string,
+  value: string,
+  chainId: number,
+) {
+  const accountAddress = await signer.getAddress();
+
+  const tx = {
+    from: accountAddress,
+    to: PERIPHERY_ADDRESS(chainId),
+    data,
+    ...(value && value !== '0'
```

Comment:
you can pass value as a simple string

Reply:
true true, change in [c36bf33](https://github.com/Voltz-Protocol/v2-off-chain-monorepo/pull/10/commits/c36bf330f7daf2dab92b10c2d0c1ef4b5e4e1cc0)